PurchasesTests/PurchaserInfoTests.swift

```diff
@@ -117,9 +117,13 @@ class BasicPurchaserInfoTests: XCTestCase {
         expect(nonConsumables as NSSet).to(contain(["onetime_purchase"]))
     }
 
-    func testOriginalApplicationVersionNull() {
-        // TODO: why this test?
-//        expect(self.purchaserInfo!.originalApplicationVersion).to(beNil())
```

Comment:
lol. 
I think the _intent_ of the test was to check that the originalApplicationVersion was null if the key doesn't exist, so I re-added the test and made it a bit more explicit (also renamed)

Reply:
lol I prefer your version